src/app/api/organizations/route.ts

```diff
@@ -0,0 +1,24 @@
+import { NextResponse } from 'next/server';
+import { getAllOrganizations } from '@/server/actions/Organizations';
+
+// @route GET /api/organizations - Get all organizations
+export async function GET() {
+  try {
+    const organizations = await getAllOrganizations();
+
+    if (!organizations) {
+      throw new Error('Organization not found');
```

Comment:
`getAllOrganizations` should always return an array. if it's empty, that's OK, it just means that there weren't any organizations